src/Context/Swoole/SwooleContextHandler.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/** @noinspection PhpComposerExtensionStubsInspection */
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Context\Swoole;
```

Comment:
I think src/Contrib/Context/Swoole is a good place for the swoole-specific classes. We are working towards doing  a subtree split, and this may end up as its own installable package.